src/Centreon/Domain/PlatformInformation/PlatformInformation.php

```diff
@@ -228,7 +228,7 @@ private function checkPortConsistency(?int $port): int
     {
         // checking
         $port = filter_var($port, FILTER_VALIDATE_INT);
-        if (false === $port || 0 > $port || $port > 65536) {
+        if (false === $port || 1 > $port || $port > 65535) {
```

Comment:
The correction is good but the error message does not explain the problem.

Reply:
Since the method property can only accept an integer or a null, the filter_var is useless.

Reply:
done